infrastructure/Taskfile.yml

```diff
@@ -223,6 +223,36 @@ tasks:
       cmds:
         - task/scripts/promote-workloads-to-prod-nodes.sh
 
+    cluster:mode:release:
+      deps: [cluster:auth , lagoon:cli:config]
+      desc: "Set the cluster in release mode by adding additional nodes to release critical node pools"
+      cmds:
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name app6
+          --resource-group rg-env-dplplat01
+          --node-count 4
+        - az aks nodepool scale
+          --cluster-name aks-dplplat01-01
+          --nodepool-name system2
+          --resource-group rg-env-dplplat01
```

Comment:
Can we move some of these hard coded values out of the task?

I did a quick search through the codebase and as far as I can tell we do not have values such as `aks-dplplat01-01`,  `app6`, `system2` or `rg-env-dplplat01` hardcoded elsewhere.

Reply:
There no values at the moment for the cluster and nodepools as TF is out for sync, additionally the way this is set up, we're missing logic for choosing the right cluster and nodepools in the cluster